Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -9591,7 +9591,7 @@
         <CargoSize value="4"/>
         <ScoreMake value="400"/>
         <ScoreKill value="800"/>
-        <SubgroupPriority value="9"/>
+        <SubgroupPriority value="15"/>

```

Comment:
Aberration - only burrow. All burrow-only should have the same priority (but still lower than Marines/Wraiths)